modules/iscsi/udiskslinuxmanageriscsiinitiator.c

```diff
@@ -282,86 +283,105 @@ handle_get_firmware_initiator_name (UDisksManagerISCSIInitiator *object,
   return TRUE;
 }
 
+
+static gchar* _get_initiator_name (GError **error)
+{
+  gchar *initiator_name = NULL;
+  gchar *initiator_contents = NULL;
+  gsize  initiator_size = 0;
+  gboolean success = FALSE;
+
+  success = g_file_get_contents (initiator_filename,
+                                 &initiator_contents,
+                                 &initiator_size,
+                                 error);
+  if (!success)
+    {
+      g_prefix_error (error, "Error reading iSCSI initiator name from '%s': ", initiator_filename);
+      return NULL;
+    }
+
+  /* We don't want to create a scanner for this iscsi initiator name grammar.
+   * So for simplicity, we just search for "InitiatorName=" prefix and if
+   * found, we just shift the name pointer.  The string may contain whitespace
+   * at the end; it's removed as well. */
+  if (strncmp (initiator_contents, initiator_name_prefix, initiator_name_prefix_len) == 0)
+    {
+      /* Shift the pointer by prefix length further. */
+      initiator_name = &initiator_contents[initiator_name_prefix_len];
```

Comment:
`initiator_contents + initiator_name_prefix_len` would be nicer here, I think.

Reply:
Fixed.